Dangerfile

```diff
@@ -8,18 +8,30 @@ warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]" or
 
 has_changelog_updates = git.modified_files.include?("CHANGELOG.md")
 if has_pod_changes && !has_changelog_updates && !declared_trivial
-  fail("Please add a line to the `CHANGELOG.md` since you changed the SDK.")
-  message("You can declare this PR as trivial by adding `\#trivial` to the PR's title.")
+  fail("Please include a CHANGELOG entry to credit yourself! \nYou can find it at /CHANGELOG.md.", :sticky => false)
+  markdown <<-MARKDOWN
+### Example CHANGELOG entry
+
+```markdown
+* **feature/improvement/bugfix** #{github.pr_title}\s\s
+  [#issue_number](https://github.com/cauliframework/cauli/issues/issue_number) by @#{github.pr_author}  
```

Comment:
The CONTRIBUTION.md does not mention that an issue should be opened, but here we propose to add an issue link. Probably a non-issue, but how do you want to handle this? Should each code change be based on an issue or are PR (and changelog entries) without issue ok? If they are ok, should we somehow mark this here as optional or provide another example without issue?

Reply:
I'm not sure if a example is necessary at all. If we already have entries in the Changelog.md, those will work as example automatically, don't they?

Reply:
True. Any thoughts regarding issue for PR vs no issue, like in the contibution guide?